io.github.tlcfem.suanPan.yml

```diff
@@ -0,0 +1,59 @@
+app-id: io.github.tlcfem.suanPan
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: suanPan
+finish-args:
+  - '--filesystem=home'
+  - '--socket=fallback-x11'
+  - '--device=dri'
+  - '--socket=wayland'
+  - '--share=ipc'
+cleanup:
+  - /vtk
+modules:
+  - name: suanPan
+    buildsystem: cmake
+    builddir: true
+    config-opts:
+      - '-DCMAKE_BUILD_TYPE=Release'
+      - '-DBUILD_MULTITHREAD=ON'
+      - '-DUSE_HDF5=ON'
+      - '-DUSE_EXTERNAL_VTK=ON'
+      - '-DVTK_DIR=/app/vtk/lib/cmake/vtk-9.2/'
+    sources:
+      - type: git
+        url: 'https://github.com/TLCFEM/suanPan.git'
+        tag: 'suanPan-v2.6.1'
+    modules:
+      - name: vtk
+        buildsystem: cmake-ninja
+        builddir: true
+        config-opts:
+          - '-DCMAKE_BUILD_TYPE=Release'
+          - '-DBUILD_SHARED_LIBS=OFF'
+          - '-DCMAKE_INSTALL_PREFIX=/app/vtk'
+        sources:
+          - type: archive
+            url: >-
+              https://www.vtk.org/files/release/9.2/VTK-9.2.2.tar.gz
+            sha256: 1c5b0a2be71fac96ff4831af69e350f7a0ea3168981f790c000709dcf9121075
+      - name: metadata
```

Comment:
it should be in the main module `suanPan`.

Reply:
I am not sure how to write it, do you mind to give me a snippet?

Reply:
like you did before but taking into account that the source dir is `..`